data-raw/phenotypes.R

```diff
@@ -69,37 +71,37 @@ phenotype_data <- phenotype_files %>%
   mutate(value = as.numeric(value)) %>%
   filter(!is.na(value))
 
-# Metadata ----
+## Metadata ----
 
-## Biospecimen metadata ----
+### Biospecimen metadata ----
 
 biospecimen_id <- "syn18876530"
 biospecimen_version <- 8
 
 check_latest_version(biospecimen_id, biospecimen_version)
 
-biospecimen_metadata_path <- synGet(biospecimen_id, version = biospecimen_version, downloadLocation = here::here("data-raw", "pathology"), ifcollision = "overwrite.local")
+biospecimen_metadata_path <- synGet(biospecimen_id, version = biospecimen_version, downloadLocation = here::here("data-raw", "pathology", "5xFAD"), ifcollision = "overwrite.local")
```

Comment:
This is fine.  I would use a configuration file to control this kind of thing though!